.github/ISSUE_TEMPLATE/reuniao.md

```diff
@@ -8,39 +8,31 @@ assignees: ''
 
 ---
 
-_Este template deve ser editado com os itens a serem discutidos durante a reunião. Toda colaboração é bem vinda. Durante a reunião devem ser registradas as participantes e as decisões tomadas, caso hajam informações sensíveis, deve ser criado um arquivo no [Google Drive da organização da Python Brasil 2021](https://drive.google.com/drive/folders/18zvyKpV42k_n_8Sr4a7yo5KxyU_SFgen?usp=sharing) e o link inserido nessa issue. Após o encerramento, o grupo responsável pela reunião deve revisar as informações e fechar a issue em não mais do que uma semana._
+_[Remover antes de abrir o ticket] Este template deve ser editado com os itens a serem discutidos durante a reunião. Toda colaboração é bem vinda. Durante a reunião devem ser registradas as participantes e as decisões tomadas, caso hajam informações sensíveis, deve ser criado um arquivo no [Google Drive da organização da Python Brasil 2021](https://drive.google.com/drive/folders/18zvyKpV42k_n_8Sr4a7yo5KxyU_SFgen?usp=sharing) e o link inserido nessa issue. Após o encerramento, o grupo responsável pela reunião deve revisar as informações e fechar a issue em não mais do que uma semana._
 
 # Reunião <Grupo de trabalho ou tema da reunião>
 
 - Data: <! –– data ––>  
 - Participantes:
   - Nome e Sobrenome (@Github ou E-mail)
 
-## Ata
-
-_Adicione os itens a serem discutidos durante a reunião. Certifique-se de que você estará presente na reunião, pois a ata só pode ser editada por quem a criou_
-
-### Acompanhamento das tarefas
+## Acompanhamento das tarefas
 
 - [ ] Verificar [tarefas bloqueadas](https://github.com/pythonbrasil/pybr2021-org/projects/1?card_filter_query=label%3Abloqueado):
   - Está claro o que precisa ser feito para desbloquear a tarefa?
   - Há parte da tarefa que pode prosseguir de alguma forma?
-- [ ] Verificar [tarefas em andamento](https://github.com/pythonbrasil/pybr2021-org/projects/1)
+- [ ] Verificar [tarefas em andamento](https://github.com/pythonbrasil/pybr2021-org/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc+-label%3ABloqueado+-label%3AP%C3%B3s-evento+project%3Apythonbrasil%2Fpybr2021-org%2F1), filtrando as bloqueadas e de pós-evento e ordenando pela últimas que foram atualizadas.
   - A tarefa está associada a uma pessoa responsável?
   - Há algo bloqueando o avanço da tarefa que possa ser discutido?
 - [ ] Verificar [tarefas abertas](https://github.com/pythonbrasil/pybr2021-org/issues?q=is%3Aopen+is%3Aissue+-project%3Apythonbrasil%2Fpybr2021-org%2F1+) não associadas ao projeto no Github.
 
 
-### Itens a serem debatidos
+## Ata
 
-Descrição | Link da Tarefa | Tempo para discussão |
-----------|--------|----------------------|
---| #x | -- |
---| #y | -- | 
---| #z | -- |
+_Adicione os itens que foram discutidos durante a reunião._
 
+- 
 
 ### Ações para serem realizadas
 
-Atividade| Responsável | Link da Tarefa | Observações|
- --| --| --|--|
+- [ ] Atividade (@responsável)
```

Comment:
esses colchetes são pra marcar se foi feito? porque entendo essa parte como ações futuras

Reply:
A minha ideia era algo assim:

https://user-images.githubusercontent.com/431892/123857693-2ba7fb80-d912-11eb-8147-2e62c58c7fa0.mov

Mas pensando melhor, dessa forma a gente não usaria o template das issues, que acaba forçando a gente a detalhar melhor a descrição. Vou mudar.